scripts/webhook-create-signed-cert.sh

```diff
@@ -30,6 +30,8 @@ done
 [ -z ${namespace} ] && namespace="default"
 [ -z ${kubectl} ] && kubectl="kubectl"
 
+which ${kubectl} > /dev/null 2>&1 || (echo "ERROR: ${kubectl} not found"; exit 1)
```

Comment:
This is should exit the whole script, but it exits only the subshell that runs what is in the parenthesis.

I'd suggest using a statement block rather than parenthesis that fork a subshell:
```
{ echo "ERROR: ${kubectl} not found"; exit 1 ; }
```

(note that the statement block needs `;` also after `exit 1`, when writing a one-liner.)